index.html

```diff
@@ -73,14 +73,14 @@ <h1 class="ddh-hero__title  hd-hr">Help improve <strong>DuckDuckGo</strong> Inst
 				<div class="ddh-hero-slides--img">
 					<div class="ddh-hero-slides__wrap">
 						<div class="ddh-hero-slides__img-wrap  js-hero-img  no-js__hide is-active">
-							<span class="ddh-hero-slides__attr  one-line"><span class="ddh-hero-slides__attr__item">Featured IA: <a href="https://duckduckgo.com/?q=regex+cheat+sheet&amp;ia=cheatsheet" target="_iaRegex">Regex</a></span>
-							<span class="ddh-hero-slides__attr__item">Contributor: <a href="https://github.com/mintsoft" target="_githubmintsoft">mintsoft</a></span>
+							<span class="ddh-hero-slides__attr  one-line"><span class="ddh-hero-slides__attr__item">Featured IA: <a href="https://duckduckgo.com/?q=%241+cad&ia=currency" target="_iaCurrency">Currency Converter</a></span>
+							<span class="ddh-hero-slides__attr__item">Contributor: <a href="https://github.com/mrchrisw" target="_githubmrchrisw">MrChrisW</a></span>
 							</span>
-							<img class="ddh-hero-slides__img  js-lazyretina" src="images/regex_hero@2x.jpg" data-src="images/regex_hero.jpg" alt="Regex Instant Answer"
+							<img class="ddh-hero-slides__img  js-lazyretina" src="images/currency_hero@2x.jpg" alt="Regex Instant Answer"
 							/>
 						</div>
 						<noscript>
-							<img class="ddh-hero-slides__img--no-js" src="images/soundcloud_hero.jpg" alt="SoundCloud Instant Answer" />
+							<img class="ddh-hero-slides__img--no-js" src="images/currency_hero@2x.jpg" alt="Currency Conversion Instant Answer" />
```

Comment:
Seeing as the default image isn't placed by JS the `<noscipt>` shouldn't actually be necessary right now. I suspect that the `ddh-hero-slides__img-wrap` is intended to be populated with JS that chooses the correct image size (1X or 2X) based on the browser. 

It seems like there's some fixing to do here but for now this should be alright 👍